frontend/src/scenes/billing/billingLogic.ts

```diff
@@ -319,6 +258,84 @@ export const billingLogic = kea<billingLogicType>([
                 router.actions.replace('/organization/billing')
             }
             actions.registerInstrumentationProps()
+
+            actions.determineBillingAlert()
+        },
+        determineBillingAlert: () => {
+            if (values.productSpecificAlert) {
+                actions.setBillingAlert(values.productSpecificAlert)
+                return
+            }
+
+            if (!values.billing || !preflight?.cloud) {
```

Comment:
looks like we need to connect preflight or otherwise load it in?

Reply:
yup, you're right. Fixed 🔨